lib/wanda_web/controllers/fallback_controller.ex

```diff
@@ -0,0 +1,19 @@
+defmodule WandaWeb.FallbackController do
+  use Phoenix.Controller
+
+  alias WandaWeb.ErrorView
+
+  def call(conn, {:error, :no_checks_selected}) do
+    conn
+    |> put_status(:unprocessable_entity)
+    |> put_view(ErrorView)
+    |> render(:"422", reason: "No checks were selected.")
+  end
+
+  def call(conn, {:error, :already_running}) do
```

Comment:
Should we have a "just in case" error here?
If for some strange reason the `DynamicSupervisor` fails to start or so.
I know that you stretched the possible errors in the spec above hehe

Reply:
done, I've also added a test that covers the generic error case.
I've omitted specific errors tests, since we cover them from the execution controller tests.